invisible_cities/detsim/light_tables_c.pyx

```diff
@@ -0,0 +1,171 @@
+import numpy  as np
+import pandas as pd
+
+cimport cython
+cimport numpy as np
+
+from libc.math cimport            sqrt
+from libc.math cimport           floor
+from libc.math cimport round as cround
+
+
+
+from ..         core import system_of_units as units
+from  . light_tables import                  read_lt
+
+cdef class LT:
+    """Base abstract class to be inherited from for all LightTables classes.
+    It needs get_values_ cython method implemented, as well as zbins_ and sensor_ids_ attributes.
+    """
+    cdef double* get_values_(self, const double x, const double y, const int sensor_id):
```

Comment:
Is there any advantage in using a raw pointer to double instead of a Memoryview?